shared-ui/components/example-button/SecondaryButton.styles.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import { max } from '../../lib/responsive';
+import { colors } from '../../style/colors';
+
+const StyledSecondaryButton = styled.button`
+  color: ${colors.BUTTON_GREEN};
+  background-color: ${colors.BUTTON_DARK_GREEN};
+  border-color: ${colors.BUTTON_GREEN};
+  padding: 0.7em;
+  border-radius: 2em;
+  cursor: pointer;
+  border: none;
+  font-size: 1em;
+
+  @media ${max.tablet} {
+    font-size: 0.7em;
```

Comment:
The buttons only look different sized on the desktop designs so let's change this to be 

`  @media ${max.tablet} {
    font-size: 0.8em;
    padding: 1em;
  };`

to match the primary button sizing on mobile